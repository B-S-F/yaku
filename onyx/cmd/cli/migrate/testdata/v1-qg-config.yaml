metadata:
  version: v1
header:
  name: External dependencies
  version: 1.16.0
autopilots:
  dummy-autopilot:
    env:
      TEST: test
    config:
      - sharepoint-fetcher-config_1.yaml
    run: |
      echo "{\"status\":\"GREEN\"}"
finalize:
  run: |
    html-finalizer
    zip-finalizer
chapters:
  "1":
    title: Title
    requirements:
      "1.1":
        title: Title
        text: Text
        checks:
          "1.1":
            title: Title
            automation:
              autopilot: dummy-autopilot
              env:
                SHAREPOINT_URL: https://sharepoint.com/project1
              config:
                - sharepoint-fetcher-config_2.yaml
      "1.2":
        title: Title
        text: Text
        checks:
          "1":
            title: Title_check
            manual:
              status: GREEN
              reason: Apes
      "1.3":
        title: Title
        text: Text
        checks:
          "1":
            title: Title_check
            manual:
              status: UNANSWERED
              reason: Apes
      "1.4":
        title: Title
        text: Text
        checks:
          "1":
            title: Title_check
            manual:
              status: UNANSWERED
              reason: Not answered
