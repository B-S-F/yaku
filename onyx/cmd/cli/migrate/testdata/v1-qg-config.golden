metadata:
    version: v1
header:
    name: External dependencies
    version: 1.16.0
autopilots:
    dummy-autopilot:
        run: |
            echo "{\"status\":\"GREEN\"}"
finalize:
    run: |
        html-finalizer
        zip-finalizer
chapters:
    "1":
        title: Title
        requirements:
            "1.1":
                title: Title
                text: Text
                checks:
                    "1.1":
                        title: Title
                        automation:
                            autopilot: dummy-autopilot
            "1.2":
                title: Title
                text: Text
                checks:
                    "1":
                        title: Title_check
                        manual:
                            status: GREEN
                            reason: Apes
            "1.3":
                title: Title
                text: Text
                checks:
                    "1":
                        title: Title_check
                        manual:
                            status: UNANSWERED
                            reason: Apes
            "1.4":
                title: Title
                text: Text
                checks:
                    "1":
                        title: Title_check
                        manual:
                            status: UNANSWERED
                            reason: Not answered
